hs_composite_resource/models.py

```diff
@@ -63,10 +63,7 @@ def get_metadata_xml(self, pretty_print=True):
     def supports_folder_creation(self, folder_full_path):
         """this checks if it is allowed to create a folder at the specified path"""
 
-        path_parts = folder_full_path.split("/")
-        # remove the new folder name from the path
-        path_parts = path_parts[:-1]
-        path_to_check = "/".join(path_parts)
+        path_to_check = folder_full_path[:folder_full_path.rfind("/")]
```

Comment:
@pkdash May need to accommodate the case when ```/``` does not exist in ```folder_full_path``` in which case path_to_check would be ```folder_full_path``` itself?